### ISSUE TITLE
save monte carlo xi in fitter2 

### ISSUE THREAD

**Issue by vserret** [igmhub/picca#410]
When we do a monte carlo simulation in fitter2 we save only the values and the errors for the fitted parameters. We should also save the MC correlation functions.

**Comment by ngbusca** [comment]
Agreed! do you want to PR it? You might need to regenerate the reference output files.

**Comment by vserret** [comment]
ok

**Comment by londumas** [comment]
@mblomqvist, it seems to me that you fixed this. Is it true? Can we close this ticket?

**Comment by mblomqvist** [comment]
@vserret fixed it. It can be closed.
